webapp/components/test-search.js

```diff
@@ -56,26 +55,28 @@ const QUERY_GRAMMAR = ohm.grammar(`
 
     CountSpecifier
       = "count" inequality number -- countInequality
-      | "count:" number           -- countEq
-      | "count=" number           -- countN
+      | "count:" number           -- countN
       | "three"                   -- count3
       | "two"                     -- count2
       | "one"                     -- count1
 
-    Exists = ListOf<RootExp, space*>
+    Exists
+      = "exists(" ListOf<Exp, space*> ")" -- explicit
```

Comment:
Looks like this adds an explicit `exists` form, can you document it in https://github.com/web-platform-tests/wpt.fyi/blob/master/api/query/README.md?